client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
I think hashing is pointless for this actually, so we should just do a string concatenation for this.